### ISSUE TITLE
Widget/Card: Add setting to remove the borders

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise-wc#1169]
**Describe the bug**
Add a setting to cards so that the border and box shadow will be hidden. Needs some answers on https://jira.infor.com/browse/ICE-85 before starting

**Expected behavior**
New class "border-less" that hides it

**Additional context**
https://jira.infor.com/browse/ICE-85

**Comment by janahintal** [comment]
Cards still has borders inside when in Classic theme in all modes. 
https://main-enterprise.demo.design.infor.com/components/cards/test-borderless.html?theme=classic&mode=contrast&colors=default
![image](https://user-images.githubusercontent.com/45757016/228002911-84915d7b-a321-4715-9b84-dce32b19b5bf.png)

**Comment by yohannahbautista** [comment]
Hello, checking in contrast, it's still borderless. The contrast is with the background
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/85279127/228120430-59064aa2-21a3-4afa-b89a-d2850d9d4d73.png">

**Comment by tmcconechy** [comment]
@yohannahbautista i think this should be a new issue https://github.com/infor-design/enterprise/issues/7374 see there..

**Comment by janahintal** [comment]
QA Passed. Now working as expected in main. Tested on All versions, mode and colors. 
![image](https://user-images.githubusercontent.com/45757016/229864255-189226ba-b1b7-445f-b166-7d4c339488b8.png)
![image](https://user-images.githubusercontent.com/45757016/229864360-93270216-f2ac-46a1-825d-6fd78aa48d31.png)
![image](https://user-images.githubusercontent.com/45757016/229864442-f34b1fe8-c8d9-4af6-8cd3-f43a6f1f326b.png)
